public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+  Before diving deeply into the details, here is a short overview of these techniques:
+  
+  __#1: Parent to child communication with input property binding__
+  
+  The `prop-value` expression of the parent is bound to the `child-prop` input property of the child. `child-prop` is updated every time when `prop-value` changes:
+  
++makeExample('component-communication/ts/fragments/property-binding-intro.html')
+  
+:marked
+  We can bind multiple properties of a single child component to the parent, and, implicitly, we can bind multiple children to the parent &mdash; 
+  ie. with `ng-for`:
+  
++makeExample('component-communication/ts/fragments/property-binding-intro-2.html')
+  
+:marked
+  [Jump to details](#-1-parent-to-child-communication-with-input-property-binding)
+
+:marked
+  __#2: Child to parent communication with event emitting__
+  
+  When a child needs to communicate with its parent, it can emit an event. The parent can define  its own event handler method , and bind this method 
+  to the child's output property:
+
++makeExample('component-communication/ts/fragments/event-binding-intro.html')
+  
+:marked
+  Here, the child event is emitted by the `raiseMyEvent` method. The parent declares the `handleMyEvent()` method and binds it to the `child-event`
+  output property of the child. Events can have arguments &mdash; here, `"Fired!"` is the argument &mdash;, which can be passed to the parent.
+  At the parent side `$event` represents the argument of the emitted event.
+  
+  [Jump to details](#-2-child-to-parent-communication-with-event-emitting)
+
+  __#3: Child to parent communication with direct method calling__
+  
+  Nothing prevents us to implement our parent so that it passes a reference to itself to its children, and those use that reference
+  to access their parent's methods and properties. When the parent embeds its children as content markup, we can hand the parent reference to 
+  children in a trivial way like this:
+  
++makeExample('component-communication/ts/fragments/parent-reference-1.html')
+  
+:marked
+  Beside this very technique, there are other ways to pass the parent reference to children, with little tricks we can even use dependency injection
+  to inject the parent into the child component's constructor.
+  The concrete technique does not matter: **Angular discourages this model, because it can sidestep the change detection mechanism, and so it may break
+  the data flow model.**
+  
+  [Jump to details](#-3-child-to-parent-communication-with-direct-method-calling)
+
+  __#4: Duplex parent/child communication through a service object__
+  
+  Parent and child components share their responsibilities. In this sharing both component types have their own responsibilities including UI logic
+  and others. The parent and its children both can manage state information about the UI logic. Often, the majority of communication is about
```

Comment:
"The parent and child delegate shared state management to the service and communicate about that state through the service."

A bit abstract. Need to think of a way to make this compelling in a few words.